packages/browser/src/Component/Itowns/3DTiles/3DTilesUtils.js

```diff
@@ -1,4 +1,145 @@
 import * as THREE from 'three';
+import { TilesManager } from './TilesManager';
+
+/**
+ * Create a geometric outliers for each tile of a tileset when its geometry is loaded
+ * Each outlier tile geometry does not keep the batchid for each outliers.
+ * May be slow to create / load.
+ *
+ * @param {TilesManager} tilesManager A TilesManage object from UD-Viz.
+ */
+export function createOutliersOfTileset(tilesManager) {
+  tilesManager.addEventListener(
+    TilesManager.EVENT_TILE_LOADED,
+    function (tile) {
+      if (tilesManager.tiles[tile.tileId] != undefined) {
+        if (tilesManager.tiles[tile.tileId].cityObjects != undefined) {
+          const geom =
+            tilesManager.tiles[tile.tileId].getObject3D().children[0]
+              .children[0];
```

Comment:
Maybe use `const geom = tilesManager.tiles[tile.tileId].getMeshes()[0];` instead of `...getObject3D().children[0].children[0]`

Since `const geom = ...getMeshes()[0];` will apply the outline only to the first mesh, you could also do `const meshes = ...getMeshes();` then loop on meshes